rados/command.go

```diff
@@ -85,50 +72,24 @@ func (c *Conn) PGCommandWithInputBuffer(pgid []byte, args [][]byte, inputBuffer
 func (c *Conn) pgCommand(pgid []byte, args [][]byte, inputBuffer []byte) ([]byte, string, error) {
 	name := C.CString(string(pgid))
 	defer C.free(unsafe.Pointer(name))
+	ci := cutil.NewCommandInput(args, inputBuffer)
+	defer ci.Free()
+	co := cutil.NewCommandOutput(radosBufferFree)
```

Comment:
Instead of passing `radosBufferFree`, I think it is easier to understand when there is a `co.SetFreeFunc()` method or the like. The current notation confused me, and I assumed `radosBufferFree` was a (global?!) variable where the output gets stored.

Reply:
Hmmm. This is a case where I miss keyword args like Python has. Something like `cutil.NewCommandOutput(freeFunc=radosBufferFree)` is nice and clear imo.
I'm not sure I like being able to construct a CommandOutput without a free function. The current style makes it mandatory.
I suppose we could default to C.free internally, and then support some chaining like `cutil.NewCommandOutput().SetFreeFunc(radosBufferFree)` ?

Reply:
Yes, default to `C.free()` with an option to change it.

Reply:
done.